tests/test_data_processing.py

```
@@ -965,306 +980,299 @@ def test_create_batch_dict_structure(self, processor):
     @pytest.fixture
     def processor(self, mock_config, mock_tokenizer, mock_words_splitter):
         """Create UniEncoderSpanDecoderProcessor instance."""
         from gliner.data_processing import UniEncoderSpanDecoderProcessor
-        
+
         decoder_tokenizer = Mock()
         decoder_tokenizer.unk_token = "[UNK]"
         decoder_tokenizer.pad_token = "[PAD]"
-        
+
         def mock_decode_tokenize(texts, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)